ResourceStatusSystem/PolicySystem/Actions/SlackAction.py

```diff
@@ -0,0 +1,122 @@
+""" SlackAction
+
+  This action posts the status change to a Slack channel
+  It uses Slack Webhooks.
+  Also compatible with Mattermost, which is an open source alternative to Slack.
+
+  To create a webhook URL refer the following :
+  Slack : https://api.slack.com/incoming-webhooks
+  Mattermost : https://docs.mattermost.com/developer/webhooks-incoming.html
+
+  Add the webhook URL to dirac.cfg at Operations/[]/ResourceStatus/Config/Slack
+
+  example:
+  Operations/
+    Defaults/
+      ResourceStatus/
+        Config/
+          Slack = https://hooks.slack.com/services/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
+          Mattermost = https://mattermost.web.cern.ch/hooks/axy94k3m1pg5xeyaw3qqb3x8bo
+
+  Even if using Mattermost,the URL is still to be placed at 
```

Comment:
This is really confusing. Maybe it would be better to give this option a more generic name "ChatAppURL" or something like that?

Reply:
Actually this is not a generic Module which would support any and every chat application. 
Even if the chat app has a URL based method of posting messages, the exact format will differ a lot. 

Take a look at the `sendMessage` method which posts the message. The Payload structure is specific to slack only.

Mattermost (an open source slack alternative) is built almost exactly like slack in terms of APIs and Message syntax. Therefore it "happens" to work in the same way as Slack.